ocaml/xapi/xapi_host.ml

```diff
@@ -683,6 +683,12 @@ let restart_agent ~__context ~host:_ =
   debug "Created process with pid: %d to perform xe-toolstack-restart"
     (Forkhelpers.getpid pid)
 
+let restart_agent ~__context ~host:_ =
```

Comment:
I'm fine with the overall solution but would suggest to no create a new function but to keep it within the existing `restart_agent`